examples/shop_app_example/lib/main.dart

```diff
@@ -82,6 +83,7 @@ class MyApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
+      scrollBehavior: WebScrollBehavior(),
```

Comment:
ShopAppExample used for web and mobile both 
Is WebScrollBehavior solution used only for web ? Maybe we can set scrollBehavior only for web builds by kIsWeb ?
```dart
return MaterialApp(
      scrollBehavior: kIsWeb ? WebScrollBehavior() : null,
```

Reply:
Sure, makes sense. I added your suggestion.